include/libopencm3/sam/4l/pm.h

```diff
@@ -26,63 +26,53 @@
 
 /* Main Clock Control MCCTRL Read/Write */
 #define PM_MCCTRL			MMIO32(PM_BASE + 0x000)
+#define PM_MCCTRL_KEY		(PM_UNLOCK_KEY)
 
-/* CPU Clock Select CPUSEL Read/Write */
-#define PM_CPUSEL			MMIO32(PM_BASE + 0x004)
```

Comment:
if you have a branch "my-feature-i-want-pull-request" and you rebase it to "fix" comments.  you just push -f that branch.  github updates the pull request, and a lot of the comments become "xxxx commented on an outdated diff".  You _can_ just make new PRs for each one, that's also totally ok.  Witness kernel lists with "patch v5" and the entire set of patches mailed out again.  Either is fine with me, but I _do_ want to see commits that are not "fixed" by later commits in the same series.